experiments/tf_trainer/tf_hub_classifier/run.py

```diff
@@ -0,0 +1,164 @@
+"""Experiments with Toxicity Dataset"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import multiprocessing
+
+# Import common flags and run code. Must be imported first.
+from tf_trainer.common import model_trainer
+
+from tf_trainer.common import dataset_input
+from tf_trainer.common import serving_input
+from tf_trainer.common import types
+from tf_trainer.tf_hub_classifier import model as tf_hub_classifier
+
+import tensorflow as tf
+
+from typing import Dict
+
+FLAGS = tf.app.flags.FLAGS
+
+tf.app.flags.DEFINE_string("text_feature_name", "comment_text",
+                           "Feature name of the text feature.")
+tf.app.flags.DEFINE_string("key_name", "comment_key",
+                           "Name of the key feature for serving examples.")
+tf.app.flags.DEFINE_integer("batch_size", 32,
+                            "The batch size to use during training.")
+tf.app.flags.DEFINE_integer("train_steps", 40000,
+                            "The number of steps to train for.")
+tf.app.flags.DEFINE_integer("eval_period", 500,
+                            "The number of steps per eval period.")
+tf.app.flags.DEFINE_integer("eval_steps", 50,
+                            "The number of steps to eval for.")
+
+# TODO: Missing fields are not handled properly yet.
+LABELS = {
+    "frac_neg": tf.float32,
+    #"frac_very_neg": tf.float32
+}  # type: Dict[str, tf.DType]
+
+
+class TFSimpleRecordInput(dataset_input.DatasetInput):
```

Comment:
Optional, but I'd say this deserves it's own class in common and then next step can be a refactoring of the existing input class to use it.

Reply:
Done.